bpaingest/projects/stemcells/contextual.py

```diff
@@ -308,6 +308,7 @@ class StemcellsProteomicsContextual(object):
     def __init__(self, path):
         xlsx_path = one(glob(path + '/*.xlsx'))
         self.sample_metadata = self._package_metadata(self._read_metadata(xlsx_path))
+        logger.debug('xlsx path is: %s', xlsx_path)
```

Comment:
debug log shouldn't be in commit

Reply:
@grahame what have I missed here? why?

Reply:
Usually I'd use % too - might be context-switching too much from other projects here :P